frontend/src/components/Login.tsx

```diff
@@ -13,11 +13,16 @@ const tailLayout = {
 const Login = () => {
     const onFinish = async (values: any) => {
         try {
-            if(!values.confirmed) {
-                console.log('Success:', values);
-                const users = await axios.get("http://localhost:3000/users", values)
-                console.log(users)
+            const user = await axios.get(`http://localhost:3000/users`, {
+                params: {
+                    password: values.password,
+                    email: values.username
+                }
+            })
+            if(user.data !== undefined ) {
+                window.location.href = `http://localhost:3200/home`;
```

Comment:
Use `useHistory` hook from `react-router`.

Reply:
Yeap, I was also thinking about, changing this method with the useHistory hook.